scripts/docker-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+pushd `dirname $0` > /dev/null
+ST2DOCSROOT="${PWD%/*}"
+popd > /dev/null
+
+docker build -t st2/st2docs -f $ST2DOCSROOT/Dockerfile $ST2DOCSROOT
```

Comment:
@LindsayHill absolutely possible with something like this: 

```
+docker run --rm -it -p 127.0.0.1:8000:8000 \
 +  -v "$ST2DOCSROOT"/docs/source:/st2docs/docs/source st2/st2docs  /bin/bash -c "make .cleandocs; make bwclivedocs"
```
We can add wrapping scripts, but I guess @lakshmi-kannan is deliberately keep all BWC (non-community stuff) existing, but not documented on community doc repo.